nixos/modules/services/audio/botamusique.nix

```
@@ -0,0 +1,114 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.botamusique;
+
+  format = pkgs.formats.ini {};
+  configFile = format.generate "botamusique.ini" cfg.settings;
+in
+{
+  meta.maintainers = with lib.maintainers; [ hexa ];
+
+  options.services.botamusique = {
+    enable = mkEnableOption "botamusique";
```

Do you mind expanding this description, usually nice to give a concise description of the package and what enabling the service does
```suggestion
    enable = mkEnableOption "Enable botamusique service to play youtube or sound on mumble.";
```

---

That is redundant: https://github.com/NixOS/nixpkgs/blob/master/lib/options.nix#L92-L99

Usually you just pass the service name to mkEnableOption

---

Updated, thanks!

---

Oh didn't realize that it constructed the description. sorry!